Tests/AppcuesKitTests/Actions/AppcuesCloseActionTests.swift

```diff
@@ -20,7 +20,7 @@ class AppcuesCloseActionTests: XCTestCase {
 
     func testInit() throws {
         // Act
-        let action = AppcuesCloseAction(configuration: AppcuesExperiencePluginConfiguration(nil))
+        let action = AppcuesCloseAction(configuration: AppcuesExperiencePluginConfiguration(nil, appcues: appcues))
```

Comment:
ah yes, my little shortcut of just using a default nil value for `appcues` in the test convenience init was not too helpful for you on these close actions 😬 thanks for cleaning up